tools/accuracy_checker/accuracy_checker/annotation_converters/lmdb.py

```diff
@@ -0,0 +1,72 @@
+"""
+Copyright (c) 2018-2021 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import lmdb
+import numpy as np
+import cv2
+
+from .format_converter import DirectoryBasedAnnotationConverter, ConverterReturn
+from ..representation import CharacterRecognitionAnnotation
+from ..config import BoolField
+
+
+class LMDBConverter(DirectoryBasedAnnotationConverter):
+    __provider__ = 'lmdb_database'
```

Comment:
your converter creates task specific annotation, while lmdb can be used for different targets.
so better to explicitly specify task in provider
```suggestion
    __provider__ = 'lmdb_text_recognition_database'
```

Reply:
Agree. Renamed.